src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Controller/EntityAclController.php

```diff
@@ -65,6 +65,7 @@ public function idps(Request $request, string $serviceId, string $id): Response
             $entity,
             $selectedIdps,
             $selectedIdps,
+            $this->authorizationService->getContact()
```

Comment:
It sounds like it is not the responsability of a auth service to provide contact information, but I might be wrong

Reply:
The Contact is part of the Symfony Badge. So it makes kind of sense, it is only handing out the contact info of the logged in user. So this is not serving as a gateway to other Contacts.